$NetBSD: patch-doc_socket++.texi,v 1.2 2022/04/25 16:19:45 tnn Exp $

socket++.texi:2337: raising the section level of @subsection which is too low
socket++.texi:2452: raising the section level of @subsection which is too low
socket++.texi:2350: raising the section level of @subsection which is too low

--- doc/socket++.texi.orig	2002-11-25 12:31:56.000000000 +0000
+++ doc/socket++.texi
@@ -2334,7 +2334,6 @@ an @code{echo} object is a smart pointer
 
 @end table
 
-@subsection tsecho.C
 @example
 // echo server. Serves clients at port 4000.
 #include <echo.h>
@@ -2348,7 +2347,6 @@ int main ()
 @}
 @end example
 
-@subsection tcecho.C
 @example
 // echo client. Sends "mary had a litte lamb" to the server
 #include <echo.h>
@@ -2449,7 +2447,6 @@ recipient previously established through
 
 @end table
 
-@subsection tcsmtp.C
 @example
 // smtp client.
 // The president sends a message to gs4t@@virginia.edu.
